lib/app/utils/validation.rb

```diff
@@ -61,7 +61,17 @@ def self.get_resource(json, version)
       lab_results: 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-observation-lab',
       pediatric_bmi_age: 'http://hl7.org/fhir/us/core/StructureDefinition/pediatric-bmi-for-age',
       pediatric_weight_height: 'http://hl7.org/fhir/us/core/StructureDefinition/pediatric-weight-for-height',
-      pulse_oximetry: 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-pulse-oximetry'
+      pulse_oximetry: 'http://hl7.org/fhir/us/core/StructureDefinition/us-core-pulse-oximetry',
+      vitals_panel: 'http://hl7.org/fhir/StructureDefinition/vitalspanel',
```

Comment:
The vitals panel, oxygen saturation, and BMI profiles are not mentioned in the list of vital signs that are part of USCDI (http://hl7.org/fhir/us/core/general-guidance.html#us-core-data-for-interoperability-and-2015-edition-common-clinical-data-set).

Reply:
It seems reasonable to remove the ones that are not listed in that table for now.  We can turn this back on easily later if we want by updating the capability statement for the server.

Reply:
Note: we just have it not generate the sequences.  We left them in the profiles and validation because it doesn't seem like it hurts anything in case we need to re-add (though it might be cruft in the future).